Titan Intel PIO Compile Failure "dynamic STT_GNU_IFUNC symbol `strcmp'..."

The failure is described below. My first question is whether anyone has seen this before and knows how to fix it. If it is a Titan system problem, I can certainly report it to the help desk and see what the issue is.

On Titan, using:

create_newcase -compset FC5AV1C-L -res ne16_g37 -case test_newconfig_intel -mach titan -project stf006 -compiler intel

I get a failure in PIO compilation:

[imn@titan-ext7:~/ACME_MASTER/cime/scripts/test_newconfig_intel] 8-) ./test_newconfig_intel.build 
 .... determining environment variables from env_mach_specific 
COMPILER=intel
MPILIB=mpich
 .... building model executable (calling ./Buildconf/cesm_build.pl) 
    .... checking namelists (calling ./preview_namelists) 
    .... calling data prestaging  
    .... calling cesm build checks 
    .... calling cesm builds for utility libraries (compiler is intel) 
      build libraries: mct gptl pio csm_share
      Thu Aug  4 12:03:39 2016 /ccs/home/imn/acme_scratch/stf006/test_newconfig_intel/bld/intel/mpich/nodebug/nothreads/mct.bldlog.160804-120310
      Thu Aug  4 12:05:37 2016 /ccs/home/imn/acme_scratch/stf006/test_newconfig_intel/bld/intel/mpich/nodebug/nothreads/gptl.bldlog.160804-120310
      Thu Aug  4 12:05:42 2016 /ccs/home/imn/acme_scratch/stf006/test_newconfig_intel/bld/intel/mpich/nodebug/nothreads/pio.bldlog.160804-120310
**ERROR: buildlib.pio failed, see /ccs/home/imn/acme_scratch/stf006/test_newconfig_intel/bld/intel/mpich/nodebug/nothreads/pio.bldlog.160804-120310
ERROR: cat /ccs/home/imn/acme_scratch/stf006/test_newconfig_intel/bld/intel/mpich/nodebug/nothreads/pio.bldlog.160804-120310**

[imn@titan-ext7:~/ACME_MASTER/cime/scripts/test_newconfig_intel] >:O cat /ccs/home/imn/acme_scratch/stf006/test_newconfig_intel/bld/intel/mpich/nodebug/nothreads/pio.bldlog.160804-120310

...

**CMake Error at /autofs/nccs-svm1_sw/xk6/cmake/2.8.10.2/cle4.0_gnu4.7.2/share/cmake-2.8/Modules/CMakeTestCCompiler.cmake:61 (message):
  The C compiler "/opt/cray/craype/2.4.2/bin/cc" is not able to compile a
  simple test program.

  It fails with the following output:

   Change Dir: /ccs/home/imn/acme_scratch/stf006/test_newconfig_intel/bld/intel/mpich/nodebug/nothreads/pio/CMakeFiles/CMakeTmp**

[imn@titan-ext7:~/ACME_MASTER/cime/scripts/test_newconfig_intel] 8-) cat /ccs/home/imn/acme_scratch/stf006/test_newconfig_intel/bld/intel/mpich/nodebug/nothreads/pio/CMakeFiles/CMakeError.log 

...

/opt/cray/craype/2.4.2/bin/cc   -O2 -fp-model precise  -O2   -DCNL  -DNDEBUG -DMCT_INTERFACE -DHAVE_MPI -DFORTRANUNDERSCORE -DNO_R16 -DLINUX  -DCNL -DCPRINTEL  -DHAVE_SLASHPROC -I.  -I/ccs/home/imn/acme_scratch/stf006/test_newconfig_intel/bld/intel/mpich/nodebug/nothreads/include -I/ccs/home/imn/acme_scratch/stf006/test_newconfig_intel/bld/intel/mpich/nodebug/nothreads/MCT/noesmf/a1l1r1i1o1g1w1/csm_share -I/opt/cray/netcdf-hdf5parallel/4.3.3.1/INTEL/14.0/include -I/opt/cray/mpt/7.2.5/gni/mpich2-intel/14.0/include -I/opt/cray/parallel-netcdf/1.6.1/INTEL/14.0/include -I/ccs/home/imn/acme_scratch/stf006/test_newconfig_intel/bld/intel/mpich/nodebug/nothreads/include -I/autofs/nccs-svm1_home2/imn/ACME_MASTER/cime/share/csm_share/shr -I/autofs/nccs-svm1_home2/imn/ACME_MASTER/cime/share/csm_share/include      CMakeFiles/cmTryCompileExec420563501.dir/testCCompiler.c.o  -o cmTryCompileExec420563501 -rdynamic 
**/usr/bin/ld: dynamic STT_GNU_IFUNC symbol `strcmp' with pointer equality in`/usr/lib64/gcc/x86_64-suse-linux/4.3/../../../../lib64/libc.a(strcmp.o)' can not be used when making an executable; recompile with -fPIE and relink with -pie**
gmake[2]: **\* [cmTryCompileExec420563501] Error 1
gmake[2]: Leaving directory `/autofs/nccs-svm1_home2/imn/acme_scratch/stf006/test_newconfig_intel/bld/intel/mpich/nodebug/nothreads/pio/CMakeFiles/CMakeTmp'
gmake[1]: *** [cmTryCompileExec420563501/fast] Error 2
gmake[1]: Leaving directory`/autofs/nccs-svm1_home2/imn/acme_scratch/stf006/test_newconfig_intel/bld/intel/mpich/nodebug/nothreads/pio/CMakeFiles/CMakeTmp'

--- a/cime/machines-acme/env_mach_specific.titan
+++ b/cime/machines-acme/env_mach_specific.titan
@@ -102,7 +102,7 @@ setenv MPSTKZ 128M
 setenv OMP_STACKSIZE 128M
 
 if ( $COMPILER == "intel" ) then
-  setenv XTPE_LINK_TYPE dynamic
+  setenv CRAYPE_LINK_TYPE dynamic
 endif 
 if ( $?PERL ) then
   printenv

